plugins/outputs/redistimeseries/redistimeseries.go

```diff
@@ -0,0 +1,88 @@
+package redistimeseries
+
+import (
+	"github.com/go-redis/redis/v7"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+var sampleConfig = `
+  ## The address of the RedisTimeSeries server.
+  address = "127.0.0.1:6379"
+
+  # Redis ACL username
+  # username = ""
+  ## password to login Redis
+  # password = ""
+  database = 0
+  ## Optional TLS Config
+  # tls_ca = "/etc/telegraf/ca.pem"
+  # tls_cert = "/etc/telegraf/cert.pem"
+  # tls_key = "/etc/telegraf/key.pem"
+  ## Use TLS but skip chain & host verification
+  # insecure_skip_verify = false
+`
+
+type RedisTimeSeries struct {
+	Address  string `toml:"address"`
+	Username string `toml:"username"`
+	Password string `toml:"password"`
+	Database int    `toml:"database"`
+	tls.ClientConfig
+	client   *redis.Client
+}
+
+func (r *RedisTimeSeries) Connect() error {
+	r.client = redis.NewClient(&redis.Options{
+		Addr:	  r.Address,
```

Comment:
Can we please make sure given address is not empty!? If you are about it, how about making `127.0.0.1:6379` the default for address and comment the option in the config?